queries/kotlin/highlights.scm

```diff
@@ -0,0 +1,162 @@
+;;; Kotlin 
+;; Issues
+
+; mapOf with type definition AND multiple entries are not highligted correctly. Issue in parser. Eg.
+; a = mapOf<String, String>(AA.value to AA.value, AA.value to AA.value)
+
+
+(function_modifier) @type
+
+;; String
+
+(line_string_literal) @string
+(multi_line_string_literal) @string
+(line_string_literal (interpolated_identifier)) @string
+(line_string_literal (interpolated_expression (line_string_literal))) @string
```

Comment:
`(interpolated_expression) @none` ? None undoes the string highlighting for the range of the interpolation. Is this something like Python f-strings or dart $ interpolations `"three: ${ 1 + 2}"`?

Reply:
ah, nice. didn't know about `@none`. I'll look into that